### DISCUSSION TITLE
Error after pscale auth login

### DISCUSSION THREAD

**Discussion by hotWater-me** [planetscale/discussion#232] [FAQs] [answered]
$ pscale auth login 
 Error: Post "https://auth.planetscale.com/oauth/authorize_device": dial tcp 52.22.104.36:443: connect: bad file descriptor

**version**
pscale version 0.109.0 (build date: 2022-07-07T21:55:35Z commit: 8a3f3a2)

**system**
apple M1  macos12.3.1

**Comment by fooforge** [comment, marked as answer]
@hotWater-me This is an error coming from your client's network stack and usually it's a too aggressive network filter interfering. Do you perhaps have something like Little Snitch or some other network filtering tool installed, and if so, what happens if you disable it or add an exemption? Also, is this reproducible after a reboot of your client?
